thermotools/thermopack
# Reference EoSs

- Added initial GERG2008 model, and moved Rgas, and t/p limits to thermomodel
- Removed unnessecary methods
- Set Rgas when deviating from default
- Tested python interface
- Added hyperdual methods to meos class
- Initial GERG mix code
- GERG model is compiling
- Introduced GERG2008_MIX
- Removed unused code
- Fixed copy functionallity got GERG2008 MIX
- Calculate differentials of zfac in main method
- Corrected some bugs etc.
- Added mixture to notebook. Currently not working.
- First working code
- Use string lenght constants for GERG db.
- First version of nist equations
- Added mixture class
- Tested MEOS CO2
- Renamed nist -> meos
- Activate meosmix calculation
- Testing ideal gas
- Protect against log(0)
- Commented currently unused code
- Added support for MEOH
- Added more fluids and missing references
- Removed commented code
- Updated file dependencies
- Make sure critical parameters are redefined with meos equation of state
- Added missing mixing paramaters
- Added code comments
- Added get function for critical volume
- Set/get Tmin/Tmax/Pmin/Pmax in thermo_model
- Corrected entropy/enthalpy methods with t,p,z as variables
- Tested enthalpies of most fluids
- Corrected ammonia model
- Added a1 and a2 for ideal Helmholtz energy
- Removed R218 and R22
- Added bibref to reducing parameters
- Added fallback density method
- Initial code for FAKEPH with meos
- Activated fake density calculation. Not yet working properly.
- Use bracket method to to calculate difficult roots of the meos pressure function
- Rebase bugs
- Use more realistic value for sigma
- Print file beeing executed for easier debugging
- Make sure all differentials are calculated when using volume shift
- Corrected Txy code
